### PR TITLE
Join an event - creates a MeetingParticipant

### PR DESCRIPTION
**Pull request by Baggerone**

### CODE REVIEW

File: application/actions/meetings_test.go [hunk 1]
```diff
@@ -61,3 +62,76 @@ func (as *ActionSuite) Test_MeetingsList() {
...
+	type testCase struct {
+		name       string
+		inviteCode string
+		location   models.Location
+		meeting    models.Meeting
+		user       models.User
+		wantInvite models.MeetingInvite
+	}
+
+	testCases := []testCase{
```
Comment (Schparky) [inline application/actions/meetings_test.go:78]: The three test cases all follow the same path in `meetingsJoin` so they're more about testing the logic in the models functions. A test case for a bad/missing meeting might be important, for example. I'm OK with leaving this as it is, though, to get to the point where we can finish the REST conversion sooner.

File: application/actions/meetings_test.go [hunk 2]
```diff
@@ -61,3 +62,76 @@ func (as *ActionSuite) Test_MeetingsList() {
...
+			req := as.JSON("/events")
+			req.Headers["Authorization"] = fmt.Sprintf("Bearer %s", tc.user.Nickname)
+			req.Headers["content-type"] = "application/json"
+			res := req.Post(reqBody)
+
+			body := res.Body.String()
+			as.Equal(200, res.Code, "incorrect status code returned, body: %s", body)
+
+			wantContains := []string{
+				fmt.Sprintf(`"nickname":"%s"`, tc.user.Nickname),
```
Comment (Schparky) [inline application/actions/meetings_test.go:120]:
```suggestion
				fmt.Sprintf(`"created_by":{"id":"%s"`, tc.user.UUID),
```
to make sure it's actually checking the creator and not just any participant

File: application/actions/requests.go [hunk 3]
```diff
@@ -39,7 +39,7 @@ func requestsList(c buffalo.Context) error {
 	return c.Render(200, render.JSON(requestsList))
 }
 
-// swagger:operation GET /requests/{request_id} RequestsGet
+// swagger:operation GET /requests/{request_id} Requests GetRequest
```
Comment (Schparky) [inline application/actions/requests.go:42]: Good catch

**Review by Schparky — APPROVED** [review]

File: application/actions/meetings_test.go (hunk 1 above)
Comment (Baggerone) [reply application/actions/meetings_test.go:83]: :+1:

File: application/actions/meetings_test.go (hunk 2 above)
Comment (Baggerone) [reply application/actions/meetings_test.go]: :+1:

File: application/actions/requests.go (hunk 3 above)
Comment (chamberlainpj) [reply application/actions/requests.go:42]: +1

**Review by chamberlainpj — APPROVED** [review]

**Pull request merged**